app/grandchallenge/publications/tasks.py

```diff
@@ -0,0 +1,15 @@
+from celery import shared_task
+
+from grandchallenge.publications.models import Publication
+from grandchallenge.publications.utils import get_identifier_csl
+
+
+@shared_task
+def update_publication_metadata():
+    for publication in Publication.objects.all():
+        csl, new_identifier = get_identifier_csl(
```

Comment:
`get_identifier_csl` can raise a `ValueError`, and if that happens this loop will stop and the later publications will never be updated. Would you be able to wrap this call in a `try:...except...` block, and if there is a value error add a warning and continue?

Reply:
ah yes, by adding a warning, do you mean through `logger.warning('some warning')`?

Reply:
Yes that would be the preferred way to do it. Setting up logging is a bit awkward, at the top of the file you need to do something like:

```
import logging

logger = logging.getLogger(__name__)
```

Then you can use the logger in the exception handler.